api/oneinch/order.go

```diff
@@ -0,0 +1,131 @@
+package oneinch
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/big"
+	"net/http"
+
+	"github.com/svanas/1inch-sdk/golang/client/orderbook"
+	"github.com/svanas/1inch-sdk/golang/helpers/consts/contracts"
+	"github.com/svanas/ladder/api/web3"
+)
+
+func GetMakerAmount(order orderbook.OrderResponse) (*big.Int, error) {
+	i, ok := new(big.Int).SetString(order.Data.MakingAmount, 10)
+	if !ok {
+		return nil, fmt.Errorf("cannot convert %s to big.Int", order.Data.MakingAmount)
+	}
+	return i, nil
+}
+
+func GetTakerAmount(order orderbook.OrderResponse) (*big.Int, error) {
+	i, ok := new(big.Int).SetString(order.Data.TakingAmount, 10)
+	if !ok {
+		return nil, fmt.Errorf("cannot convert %s to big.Int", order.Data.TakingAmount)
+	}
+	return i, nil
+}
```

Comment:
A function called "StringToBig" would be better here and would simplify this down to one function